canonical/canonical.com
# WD-7536 LXD manage page

## Done

Build u.com/lxd/manage page

## QA

- Check https://canonical-com-1129.demos.haus/lxd/manage against [copy docs](https://docs.google.com/document/d/1doGyKYSTOsX8dHvQ6W9W2U_pHQY6yfiq1bqSAJyredQ/edit) and [design](https://www.figma.com/file/ZU2ufsTJD9tEkXPqhhDMZz/U.com---LXD-(rebranding)?node-id=1%3A3&mode=dev)

- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7536
